breaddit/tests/test_comments.py

```diff
@@ -0,0 +1,65 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from breaddit.models import Post, Comment
+
+
+class CommentTestCase(TestCase):
+
+    def test_get_returns_comments(self):
```

Comment:
test only tests for 200 response, name implies checking for comment returns. Can either change name to check for 200 or, add assertion regarding comments and a proper return of them.